promscale/installation/docker.md

```diff
@@ -0,0 +1,123 @@
+# Test Promscale using a pre-built container
+If you want to try out Promscale in a test environment before getting started,
+you can use a pre-built Docker container.
+
+<highlight type="important">
+Using Promscale from a Docker container is for local testing purposes only. Do
+not use this installation method in a production environment.
+</highlight>
+
+## Install Promscale with Docker
+Before you begin, you must have Docker installed on your local system. For
+packages and instructions, see the
+[Docker installation documentation][docker-install].
+
+<procedure>
+
+### Installing Promscale with Docker
+1.  Use Docker to create a network for Promscale and TimescaleDB:
+    ```bash
+    docker network create --driver bridge promscale-timescaledb
+    ```
+1.  Install TimescaleDB in a Docker container, and make sure the service is
```

Comment:
This step and the next one are confused. The code in the next step fits to this step, and the next step is missing its code.

Reply:
@VineethReddy02 ?

Reply:
updated.